framework/validators/EachValidator.php

```diff
@@ -128,6 +128,11 @@ private function createEmbeddedValidator($model)
      */
     public function validateAttribute($model, $attribute)
     {
+        // PHP 7.4 Typed properties not initialized
+        if(property_exists($model, $attribute) && !isset($model->$attribute)) {
```

Comment:
```suggestion
        if (property_exists($model, $attribute) && !isset($model->$attribute)) {
```

Reply:
Also, I suggest to throw exception here, like InvalidArgument or something, because it will silently validate positively everything instead.

Reply:
You mean throwing some thing  like this?


```suggestion
        if (property_exists($model, $attribute) && !isset($model->$attribute)) {
            throw new InvalidArgumentException("Uninitialized property. Please set '$attribute' to null, 0, [], ...");
        }
```

Reply:
Yes, just without suggesting anything about the setting.
OR maybe better idea - when uninitialized property is detected how about setting it to `null` explicitly. This will work as a certain BC fix and in most cases keep the original workflow. I'm not sure about the drawbacks of it though (apart from obvious).

Reply:
Just tried the idea but it fails because in the test we have 
`public array $arrayTypedProperty;`

It don't accepts null as a value (?array)

I think going for **InvalidArgument** is the best approach.

```suggestion
        if (property_exists($model, $attribute) && !isset($model->$attribute)) {
            throw new InvalidArgumentException('Uninitialized property.');
        }
```

Reply:
`is_array()` should work here and setting it to `[]`.

Reply:
Sorry I'm a bit lost.

The problem is, when we have an uninitialized typed property, no matter the type (array, string, Interface, ...), we cannot access it. So cannot perform is_array() on it. Neither we can set it to null because we don't know if the property is nullable (?array, ?string, ?Interface, ...)

Following with you idea of setting to null, what about that?

```suggestion
        if (property_exists($model, $attribute) && !isset($model->$attribute)) {
            $value = null;
        } else {
            $value = $model->$attribute;
        }
```
With the current workflow it will add an error ('{value} is invalid')

Reply:
Unless someone got better idea.